superset-frontend/src/views/CRUD/welcome/ActivityTable.tsx

```diff
@@ -79,31 +97,68 @@ const ActivityContainer = styled.div`
   }
 `;
 
+const UNTITLED = t('[Untitled]');
+const UNKNOWN_TIME = t('Unknown');
+// translation keys for last action on
+const TRANS_LAST_VIEWED = `Last viewed %s`;
+const TRANS_LAST_MODIFIED = `Last modified %s`;
```

Comment:
does this actually work? I would've guessed you'd need to put the actual strings in the `t()` functions

Reply:
Yes, I did put it into `t()` later. Just making the translation keys constants because they are used in multiple statements. I can move them closer to the actual `t()` calls to reduce confusion.

Reply:
i was more wondering if the job that pulls all the strings to be translated is smart enough to know that these strings end up in a `t()` function